code/modules/suppressions/records.dm

```diff
@@ -0,0 +1,149 @@
+/datum/suppression/records
+	name = "Records Core Suppression"
+	desc = "Each meltdown, a random ever-increasing amount of abnormalities within your containment will swap places. \
+			At even intervals, a random amount of all living creatures in the facility will be teleported a short distance, \
+			with abnormalities and ordeals seemingly teleporting in the direction of their opponent."
+	reward_text = "The manager console will be equipped with function to swap the positions of abnormalities. \
+			Attribute limits of all positions will be raised by 70."
+	run_text = "The core suppression of Records department has begun. \n\
+			Meltdown immunities are forfeit and qliphoth meltdowns will have lesser timer. \n\
+			Failed meltdowns will trigger a meltdown in another cell."
+	/// How many of abnormality cells will swap places on meltdown, by percent of current count
+	var/abno_swap_percentage = 0.4
+	/// How often teleportations happen
+	var/teleport_interval = 30 SECONDS
+	/// Minimum teleport distance for mobs
+	var/teleport_min_distance = 3
+	/// How far can mobs teleport at maximum
+	var/teleport_max_distance = 5
+	/// Minimum percentage of all valid mobs that will be teleported
+	var/teleport_min_mob_count = 0.1
+	/// Maximum percent of valid mobs to be teleported
+	var/teleport_max_mob_count = 0.2
+
+/datum/suppression/records/Run(run_white = FALSE)
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_MELTDOWN_START, .proc/OnQlipMeltdown)
+	RegisterSignal(SSdcs, COMSIG_GLOB_ABNORMALITY_SWAP, .proc/OnAbnoSwap)
+	addtimer(CALLBACK(src, .proc/TeleportLivingMobs), teleport_interval)
+
+/datum/suppression/records/End()
+	UnregisterSignal(SSdcs, COMSIG_GLOB_MELTDOWN_START)
+	UnregisterSignal(SSdcs, COMSIG_GLOB_ABNORMALITY_SWAP)
+	// EVERYONE GETS INCREASED STAT LIMIT, LET'S FUCKING GOOOOOO
+	for(var/datum/job/J in SSjob.occupations)
+		J.job_attribute_limit += 70
+	for(var/mob/living/carbon/human/H in GLOB.human_list)
+		if(!H.ckey)
+			continue
+		H.adjust_attribute_limit(70)
+		to_chat(H, span_notice("You feel more powerful than ever before!"))
```

Comment:
```suggestion
		to_chat(H, span_notice("You feel new potential unlock within you!"))
```
Previous one implies immediate power growth, which is not what happens with the attribute limit being raised.

Reply:
Sure, that works.